docs/references/api/deviceshifu-deployment.md

```diff
@@ -21,54 +21,66 @@ Deployment under the ***Shifu*** architecture is a Kubernetes-native [Deployment
+  - **name** (string)<br/>driver image of the Siemens PLC, which has to be `edgehub/plc-device:v0.0.1` (for now).
 - **spec.template.spec.containers[1].env**
-    - **PLC_ADDRESS** (string)<br/>IP address of the PLC, e.g. `192.168.0.1`.
-    - **PLC_RACK** (string)<br/>RACK value of the PLC.
-    - **PLC_SLOT** (string)<br/>SLOT value of the PLC.
-    - **PLC_CONTAINER_PORT**<br/>port of the PLC drive container, which has to be `11111` (for now).
+  - **PLC_ADDRESS** (string)<br/>IP address of the PLC, e.g. `192.168.0.1`.
+  - **PLC_RACK** (string)<br/>RACK value of the PLC.
+  - **PLC_SLOT** (string)<br/>SLOT value of the PLC.
+  - **PLC_CONTAINER_PORT**<br/>port of the PLC drive container, which has to be `11111` (for now).
 
 ## RTSP DeviceShifu Deployment
 
-For a detailed example, please refer to https://github.com/Edgenesis/shifu/tree/main/examples/rtspDeviceShifu
+For a detailed example, please refer to <https://github.com/Edgenesis/shifu/tree/main/examples/rtspDeviceShifu>
 
 - **spec.template.spec.containers[1].image**
-    - **name** (string)<br/>driver image for the RTSP camera, which has to be `edgehub/camera-python:v0.0.1` (for now).
+  - **name** (string)<br/>driver image for the RTSP camera, which has to be `edgehub/camera-python:v0.0.1` (for now).
 - **spec.template.spec.containers[1].env**
-    - **IP_CAMERA_ADDRESS** (string)<br/>IP address of the camera, `192.168.0.1`
-    - **IP_CAMERA_USERNAME** (string)<br/>user name of the camera RTSP stream.
-    - **IP_CAMERA_PASSWORD** (string)<br/>password for the camera RTSP stream.
-    - **IP_CAMERA_CONTAINER_PORT**
+  - **IP_CAMERA_ADDRESS** (string)<br/>IP address of the camera, `192.168.0.1`
+  - **IP_CAMERA_USERNAME** (string)<br/>user name of the camera RTSP stream.
+  - **IP_CAMERA_PASSWORD** (string)<br/>password for the camera RTSP stream.
+  - **IP_CAMERA_CONTAINER_PORT**
         port of the RTSP camera driver container, which has to be `11111` (for now).
+
+## LwM2M DeviceShifu Deployment
+
+For a detailed example, please refer to [https://github.com/Edgenesis/shifu/tree/main/examples/lwM2MDeviceShifuWithSecurity](https://github.com/Edgenesis/shifu/tree/main/examples/lwM2MDeviceShifuWithSecurity).
+
+- **spec.template.spec.containers[0].image**
+  - **name** (string)<br/>driver image for the LwM2M device, the test version of the driver image for the LwM2M device is `edgehub/deviceshifu-http-lwm2m:nightly`. Please use the stable version in the production environment.
+- **spec.template.spec.containers[0].ports[0].containerPort**
+  - **containerPort** (integer)<br/>port for the DeviceShifu HTTP server, set to `8080`.
+- **spec.template.spec.containers[0].env**
+  - **EDGEDEVICE_NAME** (string)<br/>name of the EdgeDevice associated with the DeviceShifu, such as `edgedevice-lwm2m`.
+  - **EDGEDEVICE_NAMESPACE** (string)<br/>namespace for the EdgeDevice corresponding to DeviceShifu, set to `deviceshifu`.
```

Comment:
In this section, we will state what makes this device different from other devices (HTTP), and if the protocol does not have any particular differences, this section can be skipped.